File: ruby/functions/returning_value.rb
#!/usr/bin/ruby2.7

def returning_boolean()
  true
end

def returning_integer()
  3 + 4
end

def returning_string()
  return "hello"
end

b = returning_boolean()
puts("boolean:",
  "b = #{b}\n\n")

i = returning_integer()
puts("integer:",
  "i = #{i}\n\n")

s = returning_string()
puts("string:",
  "s = #{s}\n\n")

File: perl/functions/returning_value.pl
#!/usr/bin/perl

use strict;
use warnings;

sub returning_integer {
  return 7;
}

sub returning_string {
  return "hello";
}

my $i = returning_integer();
print("integer:\n"
  . "i = $i\n\n");

my $s = returning_string();
print("string:\n"
  . "s = $s\n\n");
